ocs_ci/helpers/dr_helpers.py

```
@@ -1687,71 +1687,163 @@ def validate_vgrc_count():
-                    )
-                    break
+    if sync_updates_to_wait < 0:
+        raise ValueError("sync_updates_to_wait must be >= 0")
+
+    with config.RunWithAcmConfigContext():
+        previous_last_group_sync_time = initial_last_group_sync_time
+        if previous_last_group_sync_time is not None:
             logger.info(
-                "The value of lastGroupSyncTime in drpc is not updated. Retrying..."
+                "Waiting for lastGroupSyncTime to change from initial value "
+                f"{previous_last_group_sync_time}"
```

Ruff (0.15.15)

[warning] Logging statement uses f-string

(G004)